src/pages/signin/LoginForm.js

```diff
@@ -166,6 +166,7 @@ class LoginForm extends React.Component {
     render() {
         const formErrorText = this.state.formError ? this.props.translate(this.state.formError) : '';
         const serverErrorText = ErrorUtils.getLatestErrorMessage(this.props.account);
+        const hasError = !_.isEmpty(formErrorText) || !_.isEmpty(serverErrorText);
```

Comment:
`!_.isEmpty(formErrorText)`

This here is kind of redundant as this already handled if `errorText` is there errorHighlight will be shown

Reply:
Your thoughts @flodnv?

Reply:
You're right, fixed that